.github/workflows/ansible-tests.yml

```diff
@@ -0,0 +1,83 @@
+---
+name: ansible-tests
+
+on:
+  pull_request:
+    branches:
+      - "main"
+
+env:
+  MIN_PYTHON_VERSION: "3.8"
+  # arbitrarily choosing the latest stable version to run ansible-test units
+  # running 'ansible-test units' for each ansible+python version isn't necessary
+  ANSIBLE_FOR_UNIT_TESTS: "2.11"
+  # defines where ansible will look for the collection
+  # must be a sub-directory of ansible_collections/
+  COLLECTION_PATH: ansible_collections/onepassword/connect
+
+jobs:
+  ansible-sanity:
+    name: Ansible Linter (v${{ matrix.ansible }}, py${{ matrix.python }})
```

Comment:
That it is called `Ansible Linter` here threw me a bit off when looking at the `Checks` tab. I was looking for `Ansible Sanity Test`

![image](https://user-images.githubusercontent.com/1965218/158364676-00cbd97a-38e9-4ec4-9060-d710f90be2f9.png)

```suggestion
    name: Ansible Sanity Test (v${{ matrix.ansible }}, py${{ matrix.python }})
```

Reply:
Good call out! I'll make that change